### ISSUE TITLE
Implement sliding FAQ sidepanel

### ISSUE THREAD

**Issue by adamkendis** [hackforla/311-data#559]
## Action Items

- [ ] Add a sticky "Need Help?" tab on the right side of the screen. When clicked, slides out/in FAQ sidepanel. This should be visible on all pages.
- [ ] Create FAQ sidepanel. Ideally, the panel will receive a list of questions/answers via props that we can save in CONSTANTS. On click, each question should expand to show the answer.
- [ ] Sticky tab and FAQ sidepanel should both be keyboard accessible. When sidepanel is open, escape keypress should close it. The sidepanel close button and all FAQs should be tabbable.

## Resources
<img width="569" alt="Screen Shot 2020-04-23 at 4 30 40 PM" src="https://user-images.githubusercontent.com/40484278/80159532-143be880-8580-11ea-9cd9-da6f456146ac.png">
<img width="565" alt="Screen Shot 2020-04-23 at 4 30 51 PM" src="https://user-images.githubusercontent.com/40484278/80159541-1c942380-8580-11ea-9862-47cc8b42f646.png">

**Comment by pras4u** [comment]
FAQ content @ https://docs.google.com/document/d/15g8psA9-w6BCEj-JxSNuLgSf0BEqLvjXrlMFK2i8EGk/edit?usp=sharing

**Comment by johnr54321** [comment]
@danarchen Please update this ticket with new design.

**Comment by johnr54321** [comment]
UX is located in: #704

**Comment by johnr54321** [comment]
waiting on design ETA Tuesday 7/30

**Comment by adamkendis** [comment]
Issue replaced by #784.
